.gitignore

```diff
@@ -38,5 +38,5 @@ nb-configuration.xml
 # Local environment
 .env
 
-# Gradle Enterprise/Develocity
-/.mvn/.gradle-enterprise
```

Comment:
We should keep this IMO, otherwise people run the risk of uploading their pre-existing config.
I'll push a commit.